cmd/createorupdate/createorupdate.go

```diff
@@ -21,6 +21,7 @@ import (
 	v20180930preview "github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
 	"github.com/openshift/openshift-azure/pkg/fakerp"
 	"github.com/openshift/openshift-azure/pkg/util/azureclient"
+	"github.com/openshift/openshift-azure/test/util/config"
```

Comment:
seems a little strange importing test modules from here

Reply:
move to `utils/filepath/config.go` as it has a test already? @kargakis ?

Reply:
the code has been moved to `pkg/fakerp`